src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/AssessmentTransformerTest.kt

```diff
@@ -459,6 +460,15 @@ class AssessmentTransformerTest {
     }
   }
 
+  @Test
+  fun `getSortedReferralHistoryNotes correctly filters user notes`() {
+    val application = TemporaryAccommodationApplicationEntityFactory().withDefaults().produce()
+    val assessment = TemporaryAccommodationAssessmentEntityFactory().withApplication(application).produce()
+    val result = assessmentTransformer.getSortedReferralHistoryNotes(assessment, cas3Events = emptyList(), filterUserNotes = true)
+
+    assertThat(result.size).isEqualTo(0)
```

Comment:
I think we need some test notes to be filtered out (and some that are retained?)